tox.ini

```diff
@@ -37,6 +37,21 @@ environment =
     MYSQL_ALLOW_EMPTY_PASSWORD=1
     MYSQL_USER=quay
 
+[testenv:py39-unit]
+setenv =
+    PYTHONDONTWRITEBYTECODE = 1
+    PYTHONPATH={toxinidir}{:}{toxinidir}
+    TEST=true
+    SKIP_DB_SCHEMA=true
+    MARKERS="not e2e"
+    TEST_DATABASE_URI=sqlite:///data/quay_sqlite_test.db
```

Comment:
data is a directory with python code, we shouldn't create temporary files there

Reply:
I would recommend to remove test/data/test.db from git - https://github.com/quay/quay/pull/2945 - and use test/data/test.db here.